homework-9/IO homework/src/Main.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Created by Diana Bochis on 08/02/2017.
+ */
+
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.*;
+import java.text.*;
+import java.util.Date;
+
+
+public class Main  {
+
+    public static void main(String[] args) throws IOException, ParseException {
+
+        //read from the file
+        FileReader file = new FileReader("C://Users//Alex Bochis//Desktop//Athletes.csv");
+        BufferedReader reader = new BufferedReader(file);
+
+        String lines = "";
+        String unparsedFile = "";
+        String myArray[];
+
+        while ((lines = reader.readLine()) != null) {
+            unparsedFile += lines;
+        }
+        reader.close();
+        myArray = unparsedFile.split(",");
+
+        for (String item : myArray) {
+            System.out.println(item);
+        }
+
+        //unordered list with all the athletes from the file
+        List<Athlete> athletes = new ArrayList<>();
+
+       Athlete athlete1 = new Athlete(11, "Umar Jorgson", "SK", 27, 30,30-27,"xxxox","xxxxx","xxoxo" );
+       Athlete athlete2 = new Athlete(1, "Jimmy Smiles", "UK", 15, 29, 29-15, "xxoox", "xooxo", "xxxxo");
+       Athlete athlete3 = new Athlete(27, "Piotr Smitzer", "CZ", 10, 30, 30-10, "xxxxx", "xxxxx", "xxxxx");
+
+        //add athletes to our list and after evaluate them, sort them according to their time performance
```

Comment:
what you read from the file is never used. You should use that and not declare new Athlete objects.